app/assets/stylesheets/overrides.scss

```diff
@@ -10,12 +11,34 @@ $navactive: $orange;
 $tabdown: $cream;
 $font: $brown;
 
+.container {
+background-color: #fff;
+border-radius: 20px ;
+}
+
+.container:not(nav-top), .container:not(nav-bottom) {
+  margin: 10px 30px;
+}
+
+.main-title {
+  padding: 5px 10px;
+  margin: 0;
+  background-color: #fff;
+}
+
+#grad {
+  background: -webkit-linear-gradient($turq, $ltturq); /* For Safari 5.1 to 6.0 */
+  background: -o-linear-gradient($turq, $ltturq); /* For Opera 11.1 to 12.0 */
+  background: -moz-linear-gradient($turq, $ltturq); /* For Firefox 3.6 to 15 */
+  background: linear-gradient($turq, $ltturq); /* Standard syntax */
+}
+
 .five-up.nav-top {
   background-color: $navmain;
   padding: 0;
 }
 
-.five-up .active a {
+.tab-bar dd.active a {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3